### PR TITLE
MVP implementation

### LINKED ISSUE #1: Expose only `CGW` from the container
- [x] expose the CGW run in the container to the host
- [x] run the config service too and make the CGW connect to it
- [x] run hardhat (or ganache) and the transaction service pointing to it
- [x] setup the config service to return `ChainInfo` for the container instance of the tx service

### PR DESCRIPTION
**Pull request by jpalvarezl**
Basic setup with a lot of redundancy but seemingly working

Closes #1

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,7 +6,56 @@ volumes:
 services:
 
   # Safe Transaction Service
+  hardhat:
+    image: ethereumoptimism/hardhat:latest
```
Comment (rmeissner) [inline docker-compose.yml:10]: Why do we requite this? This should only be required for local testing, I was assuming we should create an example that works with external rpc nodes.

Comment (jpalvarezl) [reply docker-compose.yml:10]: Ah my understanding was that we should go for a local setup first as suggested here https://github.com/gnosis/safe-client-gateway/issues/564#issuecomment-909318311

Comment (jpalvarezl) [reply docker-compose.yml:10]: > Why do we require this?

This is not final, but the simplest way I could get a container running a hardhat node

Comment (jpalvarezl) [reply docker-compose.yml]: I have removed hardhat for now in favour of an `RPC_NODE_URL` env var

**Review by Uxio0 — APPROVED** [review]

**Comment by Uxio0** [comment]
I think it's ok to go as a first version

**Pull request merged**